t/001-management.t

```diff
@@ -1,17 +1,7 @@
-use Test::Nginx::Socket::Lua 'no_plan';
-use Cwd qw(cwd);
+use lib 't';
 
-my $pwd = cwd();
-my $apicast = $ENV{TEST_NGINX_APICAST_PATH} || "$pwd/apicast";
+use TestAPIcast 'no_plan';
```

Comment:
What is `no_plan` ?

Reply:
I don't really know. I guess it defines in what order to execute the tests. Search for `plan` on http://search.cpan.org/~agent/Test-Nginx/lib/Test/Nginx/Socket.pm

https://openresty.gitbooks.io/programming-openresty/content/ also have few mentions of it.